src/js/oCrossword.js

```diff
@@ -499,14 +503,28 @@ OCrossword.prototype.assemble = function assemble() {
 		resetButton.textContent = 'Clear answers';
 
 		this.addEventListener(resetButton, 'click', clearAnswers);
-		this.rootEl.insertBefore(resetButton, wrapper);	
+		buttonRow.appendChild(resetButton);
+
+		const toggleViewButtonAboveGrid = document.createElement('button');
+		toggleViewButtonAboveGrid.classList.add('o-crossword-mobile-toggle');
+		toggleViewButtonAboveGrid.textContent = isGridView?'List view':'Grid view';
+
+		this.addEventListener(toggleViewButtonAboveGrid, 'click', toggleMobileViews);
+		this.rootEl.insertBefore(toggleViewButtonAboveGrid, gridWrapper);
 
-		const toggleViewButton = document.createElement('button');
-		toggleViewButton.classList.add('o-crossword-mobile-toggle');
-		toggleViewButton.textContent = isGridView?'List view':'Grid view';
+		const toggleViewButtonTop = document.createElement('button');
+		toggleViewButtonTop.classList.add('o-crossword-mobile-toggle');
+		toggleViewButtonTop.textContent = isGridView?'List view':'Grid view';
 
-		this.addEventListener(toggleViewButton, 'click', toggleMobileViews);
-		this.rootEl.insertBefore(toggleViewButton, wrapper);
+		this.addEventListener(toggleViewButtonTop, 'click', toggleMobileViews);
+		buttonRow.appendChild(toggleViewButtonTop);		
+
+		const toggleViewButtonBottom = document.createElement('button');
```

Comment:
Why are there 2 toggle buttons?

Reply:
Chris wanted one above and below the the clues and the grid so that users don't have to scroll back up to change the view